packages/ui/src/machines/authenticator/actions.ts

```diff
@@ -110,3 +110,13 @@ export const handleInput = assign({
     };
   },
 });
+
+export const handleBlur = assign({
+  formValues: (context, event: AuthEvent) => {
+    const { name } = event.data;
+    return {
+      ...context['formValues'],
+      [`touched_${name}`]: true,
+    };
```

Comment:
Might as well put this logic here. But to @ericclemmons maybe something more like `formik` is needed in the future